providers/cncf/kubernetes/src/airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -1132,7 +1132,10 @@ def _build_find_pod_label_selector(self, context: Context | None = None, *, excl
             **self.labels,
             **self._get_ti_pod_labels(context, include_try_number=False),
         }
-        label_strings = [f"{label_id}={label}" for label_id, label in sorted(labels.items())]
+        label_strings = [
+            f"{label_id}={str(label) if label is not None else ''}"
```

Comment:
I wonder if we'd be better off with a helper function that consistently processes these labels instead of relying on the k8s client to empty string it for us, then we can use it in both places.

Reply:
Yeah let's pull this one out as a helper that can get those things done.